pypi_timemachine/core.py

```diff
@@ -40,43 +100,20 @@ def main(cutoff_date, port, quiet):
 
     CUTOFF = parse_iso(cutoff_date)
 
-    INDEX = requests.get(MAIN_PYPI).content
+    repo = DateFilteredReleases(
+        HttpRepository(MAIN_PYPI),
```

Comment:
Forgot to say: There is no real reason not to parameterise this now... it is something that has been asked for to support private indexes. (e.g. https://www.reddit.com/r/Python/comments/1cte019/comment/l4bh02p/ https://github.com/astrofrog/pypi-timemachine/issues/8)